tests/pubsub/testfloodsub.nim

```diff
@@ -232,7 +232,7 @@ suite "FloodSub":
 
       var nodes: seq[Switch] = newSeq[Switch]()
       for i in 0..<runs:
-        nodes.add newStandardSwitch()
+        nodes.add newStandardSwitch(secureManagers = [SecureProtocol.Noise])
```

Comment:
Lets not do this. Better parametrize the test and run them twice, once with noise once with secio.

Reply:
well it is a way to test both in less time... 😄 
I added separate noise tests in the past but they were disabled few commits after for reasons I guess "tests were slow"?